suites/pacific/rgw/tier-1-upgrade_4x_to_5x_latest.yaml

```diff
@@ -0,0 +1,412 @@
+      config-file-name: test_bucket_listing_flat_ordered_versionsing.yaml
+      timeout: 300
+- test:
+    name: Bucket Listing tests
+    desc: measure execution time for unordered listing of bucket with top level objects
+    polarion-id: CEPH-83573545
+    module: sanity_rgw.py
+    config:
+      script-name: test_bucket_listing.py
+      config-file-name: test_bucket_listing_flat_unordered.yaml
+      timeout: 300
+- test:
+    name: Bucket Listing tests
+    desc: measure execution time for ordered listing of bucket with pseudo directories and objects
+    polarion-id: CEPH-83573545
+    module: sanity_rgw.py
+    config:
+      script-name: test_bucket_listing.py
+      config-file-name: test_bucket_listing_pseudo_ordered.yaml
+      timeout: 300
+- test:
+    name: Bucket Listing tests
+    desc: measure execution time for ordered listing of bucket with pseudo directories only
+    polarion-id: CEPH-83573651
+    module: sanity_rgw.py
+    config:
+      script-name: test_bucket_listing.py
+      config-file-name: test_bucket_listing_pseudo_ordered_dir_only.yaml
+      timeout: 300
+
+# Swift basic operation
+
+- test:
+    name: Swift based tests
+    desc: RGW lifecycle operations using swift
+    polarion-id: CEPH-11019
+    module: sanity_rgw.py
+    config:
+      script-name: test_swift_basic_ops.py
+      config-file-name: test_swift_basic_ops.yaml
+      timeout: 300
+
+- test:
+    name: swift versioning copy tests
+    desc: restore versioned object in swift
+    polarion-id: CEPH-10646
+    module: sanity_rgw.py
+    config:
+      script-name: test_swift_basic_ops.py
+      config-file-name: test_swift_version_copy_op.yaml
+      timeout: 500
+
+- test:
+    name: Swift based tests
+    desc: Swift bulk delete operation
+    polarion-id: CEPH-9753
+    module: sanity_rgw.py
+    config:
+      script-name: test_swift_bulk_delete.py
+      config-file-name: test_swift_bulk_delete.yaml
+      timeout: 300
+
+# resharding tests
+- test:
+    name: Resharding tests
+    desc: Reshading test - manual
+    polarion-id: CEPH-83571740
+    module: sanity_rgw.py
+    config:
+      script-name: test_dynamic_bucket_resharding.py
+      config-file-name: test_manual_resharding.yaml
+      timeout: 500
+- test:
+    name: Resharding tests
+    desc: Reshading test - dynamic
+    polarion-id: CEPH-83571740
+    module: sanity_rgw.py
+    config:
+      script-name: test_dynamic_bucket_resharding.py
+      config-file-name: test_dynamic_resharding.yaml
+      timeout: 500
+
+
+# switch from rpm to container
+- test:
+    name: switch-from-non-containerized-to-containerized-ceph-daemons
+    polarion-id: CEPH-83573510
+    module: switch_rpm_to_container.py
+    abort-on-fail: true
+
+- test:
+    name: Upgrade containerized ceph to 5.x latest
+    polarion-id: CEPH-83573679
+    module: test_ansible_upgrade.py
+    config:
+      build: '5.x'
+      ansi_config:
+        ceph_origin: distro
+        ceph_stable_release: pacific
```

Comment:
we could avoid this setting

Reply:
Removed "build: '5.x' " . I believe we need 'ceph_origin: distro' as it needs to upgrade it to the latest build which is nonGAed